canopeum_frontend/src/pages/MapPage.tsx

```diff
@@ -17,6 +18,9 @@ import { getApiBaseUrl } from '@services/apiSettings'
 const PIN_FOCUS_ZOOM_LEVEL = 12
 const MAP_DISTANCE_ZOOM_MULTIPLIER = 20
 
+const MOBILE_WIDTH = 720
```

Comment:
IIRC, you can get the SCSS variable in JS, I'm pretty sure we do it somewhere at least once (to configure part of the MUI theme). So you can reuse `$medium-width`

Reply:
We can get the variable inside our ts component but it will come as string with static unity like 'px' at the end.That would require a string transformation, which is quite inconsistent since the variable could use a different unit. Let me know what you think.

Reply:
What about `Number.parseInt(variables.mediumWidth)`

You can still store it in a module-level constant if you want.
```suggestion
const MOBILE_WIDTH = Number.parseInt(variables.mediumWidth)
```
![image](https://github.com/user-attachments/assets/7577ca62-f006-43b9-85ac-2d7832315a5f)

Reply:
> quite inconsistent since the variable could use a different unit

breakpoints are *always* pixel based. Not worried about that :)